src/call_frame.rs

```diff
@@ -1,22 +1,16 @@
 use std::num::Saturating;
-use u256::{H160, U256};
+use u256::U256;
 use zkevm_opcode_defs::ethereum_types::Address;
 
 use crate::{state::Stack, store::InMemory};
 
 #[derive(Debug, Clone)]
 pub struct CallFrame {
-    pub heap_id: u32,
-    pub aux_heap_id: u32,
-    pub calldata_heap_id: u32,
-    // Code memory is word addressable even though instructions are 64 bit wide.
-    pub code_page: Vec<U256>,
     pub pc: u64,
     /// Transient storage should be used for temporary storage within a transaction and then discarded.
     pub transient_storage: Box<InMemory>,
```

Comment:
IIUC, the transient storage is also shared by the near calls:

> Transient memory consists of pages ([page_type](https://matter-labs.github.io/eravm-spec/spec.html#page_type)) holding data or code. Each page holds 232232 bytes; all bytes are initialized to zero at genesis.
> 
> **New pages are allocated implicitly when the contract execution starts; calling another contract allocates more pages.** Pages persist for as long as they are referenced from the live code.

Emphasis mine.